tests/TitleTest.php

```diff
@@ -496,10 +497,11 @@ public function testLanguages_detailed()
     // Primary match is to the genre listing just under the title, which this tests
     public function testGenres_multiple()
     {
-        $imdb = $this->getImdb();
+        $imdb = $this->getImdb('1375666');
```

Comment:
Could you change this to nausicaa / 0087544. It's got 4 genres and the tests already have this page downloaded

Reply:
Yes. Thanks for your suggestion. Done